velox/functions/prestosql/URLFunctions.h

```diff
@@ -98,36 +98,55 @@ FOLLY_ALWAYS_INLINE void charEscape(unsigned char c, char* output) {
   output[2] = toHex(c % 16);
 }
 
+bool isDigit(char c) {
```

Comment:
`inline` for all these.  Also change the anonymous namespace to `detail`, there should be no anonymous namespace in header file.

Reply:
done, and @Yuhta just curious why anonymous namespace in header file is not allowed ?

Reply:
It's just a convention and will trigger lint warning.  In principal headers are for sharing symbols and anonymous namespace is private.